### PR TITLE
docs(svc): doc strings typos fixed

### PR DESCRIPTION
**Pull request by jsam**

### CODE REVIEW

File: renku/service/serializers/datasets.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ class DatasetListRequest(Schema):
 
 
 class DatasetDetails(Schema):
-    """Serialize dataset to response object."""
+    """Serialize a dataset to response object."""
```
Comment (rokroskar) [inline renku/service/serializers/datasets.py]:
```suggestion
    """Serialize a dataset to a response object."""
```

File: renku/service/serializers/datasets.py [hunk 2]
```diff
@@ -150,7 +150,7 @@ class DatasetFilesListRequest(Schema):
 
 
 class DatasetFileDetails(Schema):
-    """Serialzie dataset files to response object."""
+    """Serialize dataset files to response object."""
```
Comment (rokroskar) [inline renku/service/serializers/datasets.py]:
```suggestion
    """Serialize dataset files to a response object."""
```

File: renku/service/views/datasets.py [hunk 3]
```diff
@@ -129,7 +129,7 @@ def list_dataset_files_view(user, cache):
 @use_kwargs(DatasetAddRequest)
 @marshal_with(DatasetAddResponseRPC)
 @header_doc(
-    'Add uploaded file to cloned repository.', tags=(DATASET_BLUEPRINT_TAG, )
+    'Add an uploaded file to cloned repository.', tags=(DATASET_BLUEPRINT_TAG, )
```
Comment (rokroskar) [inline renku/service/views/datasets.py]:
```suggestion
    'Add an uploaded file to a cloned repository.', tags=(DATASET_BLUEPRINT_TAG, )
```

File: renku/service/serializers/datasets.py (hunk 1 above)
Comment (jsam) [reply renku/service/serializers/datasets.py]: My linter says both forms are correct. :thinking:

Comment (rokroskar) [reply renku/service/serializers/datasets.py]: run the grammar check in word to break the tie :)

Comment (jsam) [reply renku/service/serializers/datasets.py]: I don't have to. I trust you and your years of experience of being grammar police! I applied the requested changes!

**Pull request merged**